packages/driver/cypress/integration/commands/actions/select_spec.js

```diff
@@ -65,7 +65,6 @@ describe('src/cy/commands/actions/select', () => {
         expect($select.val()).to.equal('same')
         expect($select.find('option:selected')).to.have.text('Uhura')
         expect($select[0].selectedIndex).to.equal(2)
-        expect($select[0].selectedOptions[0]).to.eql($select.find('option:selected')[0])
```

Comment:
This was actually a bug that changing to esm caught. Because the driver wasn't running in strict mode, this wasn't ever caught